documentation/modules/auxiliary/admin/kerberos/forge_ticket.md

```diff
@@ -36,5 +38,174 @@ Any system leveraging kerberos as a means of authentication e.g. Active Director
 
 There are two kind of actions the module can run:
 
-1. **Forge_Silver** - Create a Silver ticket. [Default]
-2. **Forge_Golden** - Create a Golden ticket.
+1. **FORGE_SILVER** - Forge a Silver ticket. [Default]
+2. **FORGE_GOLDEN** - Forge a Golden ticket.
+
+## Scenarios
+
+### Forge Golden ticket
+
+Golden tickets can be used for persistance in an Active Directory environment. The forged golden ticket is actually a Ticket Granting Ticket (TGT) - which can be used to request arbitrary Service tickets. This module does not connect directly to a Key Distribution Center (KDC), it instead forges its own ticket.
+
+Golden tickets can be forged using a stolen Kerberos `krbtgt` account, using a password hash in NTHASH format.
+
+For golden ticket attacks the following information is required:
+
+1. `DOMAIN` - The domain, i.e.`adf3.local`
+2. `DOMAIN_SID` This is the Security Identifier for the system, i.e. `S-1-5-21-1266190811-2419310613-1856291569`
+3. `NTHASH` - The NTHASH for the krbtgt account, i.e. `767400b2c71afa35a5dca216f2389cd9`
+4. `USER` - This username will be stored within the forged ticket, this must be a user that exists in ActiveDirectory
```

Comment:
```suggestion
```suggestion
For golden ticket attacks, the following information is required:

1. `DOMAIN` - The domain, i.e.`adf3.local`
2. `DOMAIN_SID` - This is the Security Identifier for the system, i.e. `S-1-5-21-1266190811-2419310613-1856291569`
3. `NTHASH` - The NTHASH for the krbtgt account, i.e. `767400b2c71afa35a5dca216f2389cd9`
4. `USER` - This username will be stored within the forged ticket, this must be a user that exists in Active Directory
```